packages/patternfly-4/content/design-guidelines/usage-and-behavior/empty-state/design.md

```diff
@@ -8,11 +8,11 @@ Use an empty state to educate users and guide them with positive and proactive n
 
 ![Empty state elements](./img/emptystate-elements.png)
 
-1. **Icon/image (optional)** - display associated object icons here. An icon is only recommended for full page empty states
-2. **Title** - provide a concise explanation
-3. **Body** - provide additional information that helps a user understand why the space is empty, what they can do to move forward, and the value or benefit that is gained by taking the next step
-4. **Primary button** - primary call to action
-5. **Secondary buttons** (optional) - alternative options for the user. There can be more than one secondary action.
+1. **Icon/image (optional):**  Display associated object icons here. An icon is only recommended for full page empty states
```

Comment:
@mmenestr Just realized sometimes we bold the entire "word" + : ... and sometimes we don't. Probably move the ":" outside of the bolded word?